src/dispatch/incident/flows.py

```diff
@@ -584,9 +528,6 @@ def incident_stable_flow(incident_id: int, command: Optional[dict] = None, db_se
 
     log.debug(f"We have updated the status of the external ticket to {IncidentStatus.stable}.")
 
-    # we update the conversation topic
-    set_conversation_topic(incident)
-
```

Comment:
Where are we updating the topic now? We also call this flow directly from the Slack code when run `/dispatch-mark-stable`.

Reply:
Same as above.